specs/commandline/version.md

```diff
@@ -7,7 +7,7 @@ Use `notation version` to print notation version information.
 Upon successful execution, the following information is printed.
 
 ```text
-Notation: Notary v2, A tool to sign, store, and verify artifacts.
+Notation: A tool to sign, store, and verify artifacts.
```

Comment:
Suggest removing "store" from the sentence and use the same unified sentence as `Notation - a tool to sign and verify artifacts. `